client/src/app/App.js

```diff
@@ -1946,6 +1946,11 @@ export class App extends PureComponent {
       return this.emitWithTab(type, activeTab, payload);
     }
 
+    if (action === 'toggleBottomPanel') {
```

Comment:
We already have the action `close-panel`, which closes the bottom panel.

I'd suggest we rename that action to `close-bottom-panel` and this particular action to `toggle-bottom-panel`. `camelCase` is not the pattern we use.

```suggestion
    if (action === 'toggle-bottom-panel') {
```

Reply:
I originally used `toggle-button-panel`, but since all actions in the file are in `camelCase`, I updated it to match that style.
<img width="457" alt="image" src="https://github.com/user-attachments/assets/9c4a93f0-b640-4913-997b-cdf54193222a">